sv/config.go

```diff
@@ -62,6 +63,7 @@ type VersioningConfig struct {
 // TagConfig tag preferences.
 type TagConfig struct {
 	Pattern string `yaml:"pattern"`
+	Filter  string `yaml:"filter"`
```

Comment:
What do you think?

Reply:
im ok if the filter is used on the other commands too, like `next-version`...

since it' related with this [comment](https://github.com/bvieira/sv4git/pull/45/files#r841296909), im going mark this conversation as resolved